src/PdfKit/PdfAnnotation.cs

```diff
@@ -0,0 +1,42 @@
+﻿//
+// PdfAnnotation.cs
+//
+// Authors:
+//	Alex Soto  <alexsoto@microsoft.com>
+//
+// Copyright 2017 Xamarin Inc. All rights reserved.
+//
+
+using System;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+
+#if XAMCORE_2_0 && !MONOMAC
+namespace XamCore.PdfKit {
+	public partial class PdfAnnotation {
+
+		// [Mac (10,12)] Headers do not show changes on Mac, but added on iOS 11 headers
+		public bool SetValue<T> (T value, PdfAnnotationKey key) where T : class, INativeObject
+		{
+			return _SetValue (value.Handle, key.GetConstant ());
+		}
+
+		// [Mac (10,12)] Headers do not show changes on Mac, but added on iOS 11 headers
+		public bool SetValue (string str, PdfAnnotationKey key)
+		{
+			var nstr = NSString.CreateNative (str);
```

Comment:
`setValue:forAnnotationKey:` does not have `nonnull` nor documentation says it is allowed so I think it is fine to allow it.